raiden/tests/integration/test_raidenservice.py

```diff
@@ -38,12 +37,20 @@ def test_regression_filters_must_be_installed_from_confirmed_block(raiden_networ
         to_identifier='latest',
     )
 
-    assert search_for_item(app0_state_changes, Block, {
-        'block_number': target_block_num - DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS,
-    })
-    assert not search_for_item(app0_state_changes, Block, {
-        'block_number': target_block_num,
-    })
+    assert search_for_item(
+        app0_state_changes,
```

Comment:
This was changed automatically by the flake8 hook

Reply:
What do you mean? so the code was formatted by flake8 somehow?

Reply:
yeas, I run autopep8 as a precommit hook